src/database/firebase_object_factory.spec.ts

```diff
@@ -81,12 +81,12 @@ describe('FirebaseObjectFactory', () => {
 
 
     it('should emit unwrapped data by default', (done: any) => {
-      ref.set({ data: 'bar' }, () => {
+      const fixtureData = { data: 'bar' };
+      ref.set(fixtureData, () => {
         subscription = observable.subscribe(unwrapped => {
```

Comment:
@davideast I'm not sure I follow. How can we assert the object properties after `ref.set()`, other than by subscribing to the object observable?

If it'll take too long to explain, please feel free to just tell me what to do here specifically, and I'm sure I'll figure it out as I implement the changes. :sunglasses: